src/common/snippets/src/op/subgraph.cpp

```diff
@@ -100,7 +133,8 @@ auto snippets::op::Subgraph::wrap_node_as_subgraph(const std::shared_ptr<ov::Nod
 
     for (const auto& input : node->input_values()) {
         if ((utils::is_scalar_constant(input.get_node_shared_ptr())) ||
-            (ov::is_type<ov::op::v0::FakeQuantize>(node) && ov::is_type<ov::op::v0::Constant>(input.get_node_shared_ptr()))) {
+            (ov::is_type<ov::op::v0::FakeQuantize>(node) && ov::is_type<ov::op::v0::Constant>(input.get_node_shared_ptr())) ||
+            (ov::is_type<ov::op::v1::Transpose>(node) && ov::is_type<ov::op::v0::Constant>(input.get_node_shared_ptr()))) {
```

Comment:
I think that list of ops with constant inputs is increased and will be increased. Maybe it would be better, what do you think? Just to move check for `Constant` as common check

```suggestion
if (ov::is_type<ov::op::v0::Constant>(input.get_node_shared_ptr()) &&
    (ngraph::shape_size(source_output_node->get_shape()) == 1 ||
    ov::is_type<ov::op::v0::FakeQuantize>(node) || ov::is_type<ov::op::v1::Transpose>(node) ))
```

Reply:
Yes, good suggestion.